DigitalLearningSolutions.Web/Views/Shared/MiniHub/_MiniHubHeading.cshtml

```diff
@@ -2,7 +2,7 @@
 @model MiniHubNavigationModel
 
 <h1 class="nhsuk-heading-xl">
-  <span role="text">

```

Comment:
This isn't part of the official ARIA spec and is only supported on certain browsers (that broke the accessibility test). I tried without this line and it didn't change the screenreader behaviour, so I reckon best to remove it.